smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -275,6 +370,9 @@ contract BridgeBank is BankStorage,
         // decimals defaults to 18 if call to decimals fails
         uint8 decimals = getDecimals(token);
 
+        // Denom defaults to "" (empty string) if call to cosmosDenom fails
+        string memory denom = getDenom(token);
```

Comment:
We will need to add the migration code to an IbcToken for the new rowan contract so that we can migrate from erowan to rowan and have the new rowan contract play nice with this system.